pygradle-plugin/src/main/groovy/com/linkedin/gradle/python/util/PackageInfo.groovy

```diff
@@ -96,9 +101,13 @@ class PackageInfo {
         }
     }
 
+    public static PackageInfo fromPath(Path packagePath) {
+        return fromPath(packagePath.toFile())
+    }
+
 
     @Override
     public String toString() {
-        return name + "-" + version
+        return name + "-" + version + "@" + packageFile
```

Comment:
Do we rely on this being just name-version anywhere?

Reply:
I don't think so